src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+table.querySelectorAll('tr').forEach((tr) => {
+  const clone = tr.children[1].cloneNode(true);
+
+  tr.insertBefore(clone, tr.lastElementChild);
+});
```

Comment:
Ensure that each row (`tr`) has at least two children before attempting to clone the second child. Otherwise, this could lead to an error if a row has fewer than two children.